addons/zoom-slider/userscript.js

```diff
@@ -0,0 +1,79 @@
+export default async function ({ addon, msg }) {
+  const addSlider = async () => {
+    if (addon.tab.editorMode !== "editor" || addon.self.disabled) {
+      return;
+    }
+
+    const zoom = await addon.tab.waitForElement(".blocklyZoom");
+    const blockly = await addon.tab.traps.getBlockly();
+    window.blockly = blockly;
```

Comment:
I'm pretty sure this isn't necessary? Correct me if I'm wrong.

Reply:
Oh, you're right, I put that there for debugging.